/* 
 * File:   Manager.h
 * Author: Carlos Amores (AICIA)
 *
 * Created on 13 de mayo de 2014, 12:39
 */

#ifndef MANAGER_H
#define	MANAGER_H

#ifdef	__cplusplus
extern "C" {
#endif




#ifdef	__cplusplus
}
#endif

#endif	/* MANAGER_H */

#include "ros/ros.h"
#include "CITIUS_Control_Manager/srv_status.h"
#include "CITIUS_Control_Manager/srv_electric.h"
#include "CITIUS_Control_Manager/srv_frontcam.h"
#include "CITIUS_Control_Manager/srv_rearcam.h"
#include "CITIUS_Control_Manager/srv_vehicle.h"
